extensions/gdx-freetype/src/com/badlogic/gdx/graphics/g2d/freetype/FreeTypeFontGenerator.java

```diff
@@ -598,8 +598,9 @@ public FreeTypeBitmapFontData generateData (FreeTypeFontParameter parameter, Fre
 			}
 		}
 
-		Rectangle rect = packer.pack(mainPixmap);
-		glyph.page = packer.getPages().size - 1; // Glyph is always packed into the last page for now.
+		String pixmapName = Integer.toString(glyph.hashCode());
```

Comment:
I think you should probably consider using this instead of a not-necessarily-unique hashCode():

```java
		String pixmapName = Integer.toString(glyph.id);
```

The ID should be unique in any font, barring some weirdness with alternate glyphs. The hashCode() is very likely to be unique, but it is the same as calling System.identityHashCode(), and that's neither as fast as it could be nor as unique as we want it to be.